client/App.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import { createStore, applyMiddleware } from 'redux';
 import { Provider } from 'react-redux';
 import logger from 'redux-logger';
-import Container from './app/container';
+import Container from './app/Container';
```

Comment:
Got an error here because the packager couldn't find a Container.js file, was container.js was supposed to be renamed?

Reply:
Yup, the `container.js` file should have been renamed to `Container.js`